src/main/resources/mcpi/api/python/modded/mcpi/util.py

```diff
@@ -15,4 +15,4 @@ def _misc_to_bytes(m):
 
     See `Connection.send` for more details.
     """
-    return str(m).encode("cp437")
+    return str(m).encode("UTF-8")
```

Comment:
Have you tested what would happen if you used the 'standard' mcpi python library? Does it still work?